### ISSUE TITLE
Could not get the pciture showed in email notification for issue

### ISSUE THREAD

**Issue by suli3** [go-gitea/gitea#10781]
- Gitea version (or commit ref):1.11.3
- Git version:
- Operating system: docker
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

I can get the email notification when issue @me, but for the pictures in this issue or issue reply, email notification could not show that correctly like below.

## Screenshots

![image](https://user-images.githubusercontent.com/1057425/77165375-1e844600-6aed-11ea-88cc-c3958a967bab.png)

**Comment by lunny** [comment]
Is that a private repository? 

For a private repository, we have to generate a secret for that image so that they could be displayed correctly on the emails.

**Comment by esradioli** [comment]
> Is that a private repository?
> 
> For a private repository, we have to generate a secret for that image so that they could be displayed correctly on the emails.

Yes, It is a private repository, but I am the collaborator, so this image should be displayed correctly on the email to me. So how to generate a secret for that image?

**Comment by lafriks** [comment]
It can not be displayed in email as email client is accessing it as anonymous user so there is really no easy way to fix this

**Comment by suli3** [comment]
> It can not be displayed in email as email client is accessing it as anonymous user so there is really no easy way to fix this

So, should this function be added as a part of "Reply by email"? 
I do understand it is a big function.

**Comment by lunny** [comment]
The private picture URL should be generated to resolve the problem.

**Comment by guillep2k** [comment]
@lunny you mean a `data:` type URL? I'm not sure many email clients support that.

**Comment by lunny** [comment]
@guillep2k No. I mean to generate a http query token for the private images so that the images could be reviewed on email.

**Comment by zeripath** [comment]
We could use the AvatarLink format...

**Comment by guillep2k** [comment]
> 
> 
> @guillep2k No. I mean to generate a http query token for the private images so that the images could be reviewed on email.

That's a good idea! I would tie the token to that specific resource (i.e. it would only be useful for that particular image).

**Comment by guillep2k** [comment]
> 
> 
> We could use the AvatarLink format...

Oh, yes, that would be equivalent, wouldn't it?
